diff --git a/node_modules/firebase-auth-lite/src/main.js b/node_modules/firebase-auth-lite/src/main.js
index 16c5226..0bec924 100644
--- a/node_modules/firebase-auth-lite/src/main.js
+++ b/node_modules/firebase-auth-lite/src/main.js
@@ -39,7 +39,7 @@ const storageApi = {};
  * @param {string} options.redirectUri The redirect URL used by OAuth providers.
  * @param {Array.<ProviderOptions|string>} options.providers Array of arguments that will be passed to the addProvider method.
  */
-export default class Auth {
+module.exports = class Auth {
 	constructor({ apiKey, redirectUri, name = 'default', storage = storageApi } = {}) {
 		if (!apiKey) throw Error('The argument "apiKey" is required');
 
@@ -314,7 +314,12 @@ export default class Auth {
 		// Email sign-in flow.
 		if (location.href.match(/[&?]oobCode=/)) {
 			const oobCode = location.href.match(/[?&]oobCode=([^&]+)/)[1];
-			const email = location.href.match(/[?&]email=([^&]+)/)[1];
+			let emailMatch = location.href.match(/[?&]email=([^&]+)/);
+			if (!emailMatch) {
+				const callback = decodeURIComponent(location.href.match(/[?&]continueUrl=([^&]+)/)[1]);
+				emailMatch = callback.match(/[?&]email=([^&]+)/);
+			}
+			const email = emailMatch[1];
 			const expiresAt = Date.now() + 3600 * 1000;
 			const { idToken, refreshToken } = await this.api('signInWithEmailLink', { oobCode, email });
 
